packages/core/src/common/keys.ts

```diff
@@ -53,6 +53,30 @@ export class KeyCode {
         }
     }
 
+    /**
+     * Parses a Keystroke object from a string.
+     * @param keybinding String representation of a keybinding
+     */
+    public static parseKeystroke(keybinding: string): KeyCode | undefined {
+
+        const sequence: string[] = [];
+        const keys = keybinding.split('+');
+        for (const keyString of keys) {
+            const key = CODE_TO_KEY[keyString];
+            if (Key.isKey(key)) {
+                if (Key.isModifier(key.code)) {
+                    sequence.push(MODIFIERS.filter(item => item.code === key.code)[0].code);
+                } else {
+                    sequence.push(key.code);
+                }
+            } else {
+                return undefined;
+            }
+        }
+
+        return new KeyCode(sequence.join('+'));
```

Comment:
This sounds really weird parse from string then create a keycode from another string with the modifier keys changed...
Could that be directly in Keycode ?
Like Keycode.fromUserConfig() ?